ipams/remote/remote.go

```diff
@@ -93,6 +93,8 @@ func (a *allocator) RequestAddress(poolID string, address net.IP, options map[st
 	}
 	if res.Address != "" {
 		retAddress, err = types.ParseCIDR(res.Address)
+	} else {
+		return nil, nil, ipamapi.ErrNoAvailableIPs
```

Comment:
Right but that wouldn't handle a misbehaving driver. A driver returning an empty address will cause `RequestAddr()` to return `nil` because of the conditional assignment of `redAddress`. I'm happy to fix this in another way but I don't think we should depend on a well behaving driver here.